File: stringMingling/stringMingle2.py
def mingle(stringp,stringq):
    newString = ""
    for i in range(len(stringp)):
        newString += stringq[-i-1] + stringp[-i-1]
    for i in range(len(stringp)):
        if stringp[i] == "a":
            print("La letra " +stringp[i]+ " aparece por primera vez en la posicion: "+str(i))
            
    return newString
    
stringp = "abcde"
stringq = "pqrst"
print(mingle(stringp,stringq))

File: stringMingling/stringMingle2.java
package stringMingling;

public class stringMingle2 {
    
    public static String stringMingle(String stringp, String stringq){
        String newString = "";
        for(int i=stringp.length()-1;i>=0;i--){
            newString += stringq.charAt(i);
            newString += stringp.charAt(i);
        }
        return newString;
    }

    public static void main(String[] args) {
        String stringp = "abcde";
        String stringq = "pqrst";
        System.out.println(stringMingle(stringp, stringq));
    }
}
